be/src/storage/tablet_updates.cpp

```diff
@@ -1012,8 +1014,8 @@ void TabletUpdates::do_apply() {
         if (config::enable_retry_apply && _is_retryable(apply_st) && !apply_st.ok()) {
             //reset pk index, reset rowset_update_states, reset compaction_state
             _reset_apply_status(*version_info_apply);
-            auto time_point =
-                    std::chrono::steady_clock::now() + std::chrono::seconds(config::retry_apply_interval_second);
+            size_t interval_seconds = std::min((size_t)1800, config::retry_apply_interval_second * _apply_failed_time);
```

Comment:
is 1800 seconds too long?

Reply:
> is 1800 seconds too long?

1800s is the maximum retry interval seconds. Do you think 600s would be better?